src/daily_output.ts

```diff
@@ -66,13 +68,37 @@ export class DailyOutput {
 	 * - group effort rewards, and
 	 * - streak rewards
 	 */
-	private finishDataAggregation(): void {
+	private async finishDataAggregation(): Promise<void> {
+		// check for previous unbundled bundles
+		const prevNonUnbundledBundles: { txId: string; tip: number }[] = existsSync('non_unbundled_bundles.json')
```

Comment:
Let's extract that file name to a constant

Reply:
cbcfb0e571c7e90691ccb81d772ade4edaf41451